src/http/interceptors.ts

```diff
@@ -45,9 +50,23 @@ export const camelizeResponseData = ({
     ['arraybuffer', 'stream'].includes(config.responseType ?? '') ||
     config?.interceptors?.skipCamelizeResponseData
       ? data
-      : camelizeKeys(data),
+      : camelizeKeys(
+          data,
+          config?.normalizeCustomPropertyCasing ? [] : config?.customProperties
+        ),
 });
 
+export const decamilizeRequestData = (
```

Comment:
I think this should be `decamelizeRequestData` and also should it be `snakifyRequestData` or `snake'nmakeRequestData`.... 🐍

Reply:
Create a ticket for this version to rename `decamelize` -> `snakify`

Reply:
https://extrahorizon.atlassian.net/browse/SVCS-2573